CHANGELOG.md

```diff
@@ -1,12 +1,30 @@
 # Changes in Fbx Exporter
 
+## [3.2.0-preview.1] - 2020-04-28
```

Comment:
Please review the CHANGELOG. The rest of the changes have passed a code review (feel free to look over them again though).